plutus-tx/src/Language/PlutusTx/Ratio.hs

```diff
@@ -182,23 +246,24 @@ truncate r = let (m, _ :: Rational) = properFraction r in m
 properFraction :: Ratio Integer -> (Integer, Ratio Integer)
 properFraction (n :% d) = (q, r :% d) where (q, r) = quotRem n d
 
-{-# INLINABLE quot #-}
-quot :: Integer -> Integer -> Integer
-quot x y = let (q, _ :: Integer) = quotRem x y in q
+{-# INLINABLE divMod #-}
+-- | simultaneous div and mod
+-- TODO. This is doing twice as much work as it needs to: the Plutus Core
```

Comment:
I think that's fine. Also don't put it in the Haddock, you can stick this above.

Reply:
Thanks: I didn't notice that it was inside the the Haddock comment.